src/api.js

```diff
@@ -35,4 +35,10 @@ export const postComment = (review_id, comment) => {
     })
 }
 
+export const getReviewsByCategory = (sortByQuery) => {
+    return gamesApi.get("/api/reviews").then((res)=>{
+        const reviews= res.data.reviews
+        return reviews.filter(review=>review.category===sortByQuery)
+    })
```

Comment:
we want the api to return the correct reviews, not to filter them out in the front end.

If you hard coded the permitted category values in your back end, you likely only selected the test data categories and the actual data has more categories. If this is why you did this then you'll have to go in your back end and just add these extra categories to your validated list. 

Once you do the above, you should be able to use the getReviews function and pass it a param of category 👍